custom-files/Bot Sharder/sharded-bot.js

```diff
@@ -0,0 +1,100 @@
+// Made by TheMonDon#1721
+// Some code by General Wrex
+const version = '1.1';
+
+// Include discord.js and original check
+const { version: djsVersion, ShardingManager } = require('discord.js');
+if (djsVersion < '12.0.0') {
+  console.log(
+    'This version of Discord Bot Maker requires Discord.JS v12.\nPlease use "Project > Module Manager" and "Project > Reinstall Node Modules" to update to Discord.JS v12.',
+  );
+  throw new Error('Need Discord.JS v12 to Run!!!');
+}
+
+console.log('-'.repeat(50));
+console.log("TheMonDon's DBM Bot Sharder");
+console.log(`Version: ${version}`);
+console.log("You can change the amount of shards by providing '--shard_count=[number]' (default: auto)");
+console.log("You can change the bot file by providing '--startup=./index.js' (default bot.js)");
```

Comment:
Why do you put index.js instead of placeholders like [file]

Reply:
Just to show that it requires './[file]', though could've done that. Not much of a difference.